src/inmanta/server/services/metricservice.py

```diff
@@ -34,9 +34,10 @@
 class MetricsService(protocol.ServerSlice):
     """Slice managing metrics collector"""
 
-    def __init__(self) -> None:
+    def __init__(self, extra_tags: dict[str, str] = {"component": "server"}) -> None:
```

Comment:
You have a mutable default here. If typed as `Mapping`, we have some confidence that it won't cause trouble, but even then it might be considered bad practice.